Bootstrap/Podfile.lock

```diff
@@ -35,10 +34,10 @@ CHECKOUT OPTIONS:
     :git: https://github.com/facebook/ios-snapshot-test-case.git
 
 SPEC CHECKSUMS:
-  FBSnapshotTestCase: 094f9f314decbabe373b87cc339bea235a63e07a
+  FBSnapshotTestCase: '094f9f314decbabe373b87cc339bea235a63e07a'
```

Comment:
but shouldn't all lines that I've touched be in quotes? 🕵️ 

anyway, using `ruby 2.3.3p222 (2016-11-21 revision 56859) [x86_64-darwin16]` here 😂